src/main/java/com/bitwig/extensions/controllers/arturia/keylab/mk2/ArturiaKeylabMkII.java

```diff
@@ -506,6 +511,87 @@ private void initMultiLayer()
       });
    }
 
+   private void initNotificationLayer()
+   {
+      mNotificationLayer.showText(this::getNotificationTopLine, this::getNotificationBottomLine);
+
+      for (final RelativeHardwareControl encoder : mEncoders)
+      {
+         encoder.isUpdatingTargetValue().markInterested();
+         encoder.targetName().markInterested();
+         encoder.targetDisplayedValue().markInterested();
+         encoder.isBeingTouched().markInterested();
```

Comment:
Is this line strictly necessary? I don't see any use of the isBeingTouched value anywhere. Or am I just missing it?